src/instruments/src/EFB/Settings/Settings.tsx

```diff
@@ -249,7 +249,10 @@ const SimOptionsPage = () => {
 
     const [dynamicRegistration, setDynamicRegistration] = usePersistentProperty('DYNAMIC_REGISTRATION_DECAL', '0');
 
-    const adirsAlignTimeButtons: (ButtonType & AdirsButton)[] = [
+    const [realisticTiller, setRealisticTiller] = usePersistentProperty('REALISTIC_TILLER_ENABLED', '0');
+    const [, setRealisticTillerSimVar] = useSimVar('L:A32NX_REALISTIC_TILLER_ENABLED', 'number', 0);
```

Comment:
This shouldn't be necessary as the `sync.ts` subscribes to the NXDataStore (/persistentProperty), keeping the simvar up-to-date as the setting is changed.

Reply:
You mean removing 
const [, setRealisticTillerSimVar] = useSimVar('L:A32NX_REALISTIC_TILLER_ENABLED', 'number', 0);
?

Reply:
My fault, I'm dumb :D will fix in a moment

Reply:
If you understand the issue you are waaaay less dumb tha me :D